src/v3/tsconfig.base.json

```diff
@@ -11,12 +11,34 @@
     "moduleResolution": "node",
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
+    "checkJs": false,
```

Comment:
same as `allowJs: true`?

Reply:
This was to prevent TS from complaining on v2 TS files, I dont think I need this anymore since I added the ts-nocheck to the two files that prevented tests from running. But it works in tandem with allowJS. I will remove and re-run locally to ensure this isnt needed and will delete if that is the case.